src/sugar3/graphics/radiotoolbutton.py

```diff
@@ -29,7 +35,27 @@
 
 
 class RadioToolButton(Gtk.RadioToolButton):
-    """An implementation of a "push" button."""
+    '''
+    An implementation of a "push" button. Properies that can
+	be set are:
+		accelerator (string)
+		tooltip (string)
+		xocolor (:class:`XoColor`)
+		hide_tooltip_on_click (bool)
+    
+    Args:
+        icon_name (string): name of icon to be used
+		
+		kwargs can be used to set the following properties
```

Comment:
You can document kwargs by just using the Keyword Args: header and then using the same syntax as normal args